community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        
+        apiVersion: pingcap.com/v1alpha1
```

Comment:
This field should be formatted as a json list, rather than yaml.

Reply:
Fixed, PTAL. Thank you.